pkg/registry/registry.go

```diff
@@ -114,7 +114,7 @@ func GetHiddenExposePorts(image string) []model.Port {
 	var err error
 	var username string
 	var token string
-	if okteto.IsOktetoContext() {
+	if okteto.IsOktetoContext() && strings.HasPrefix(image, okteto.DevRegistry) {
```

Comment:
why do we need this?

Reply:
We need that for getting expose fields from public docker hub images. Compose test was failing because we couldn't retrieve mongo and rabbit ports and we were not creating any service for those services

Reply:
I just realized that we need to add that it can start with the global tag